### ISSUE TITLE
`bind_key` docstring incorrect

### ISSUE THREAD

**Issue by ianhi** [napari/napari#2355]
## 📚 Documentation
The docstring contains a parameter `keymap` that is not part of the function signature
![image](https://user-images.githubusercontent.com/10111092/110063302-e21b1b80-7d38-11eb-93bb-3a175f5cfc78.png)

**Comment by sofroniewn** [comment]
Good catch, I think we should just drop from doc string. Do you mind submitting a PR? Thanks!!

**Comment by ianhi** [comment]
Looking at this deeper i'm not sure that just dropping it from the docstring is correct. 
The actual base function does have `keymap` as an arugment
https://github.com/napari/napari/blob/1b52b2bfe4337f10163c1f13587f1f66a71a9474/napari/utils/key_bindings.py#L185-L191 It But it looks as though the `viewer.bind_key` function is generated by this:
https://github.com/napari/napari/blob/1b52b2bfe4337f10163c1f13587f1f66a71a9474/napari/utils/key_bindings.py#L272-L276

so perhaps class should also be responsible for eliminating the keymap?

Or maybe a simpler thing to do is to provide the function via a closure, something like?
```python
def gen_bind_key(keymap):
    def bind_key(key, func):
       ....
    return bind_key
```

**Comment by sofroniewn** [comment]
Ah ok, we're actually going to be doing some refactoring of this fairly soon to simplify it a little an unify some things in an action manager, so maybe we wait here a little till we've gotten further with that - cc @Carreau for visibility
